webpack.config.js

```diff
@@ -23,7 +23,6 @@ module.exports = opts => {
 
     resolve: {
       modules: [
-        `${__dirname}/client/vendor`,
```

Comment:
Let's remove `resolve.modules` completely as `node_modules` are used by default.

Reply:
Good point, I've just pushed the update.